src/actions/loginActions/__tests__/loginAction.test.js

```diff
@@ -0,0 +1,83 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import fetchMock from 'fetch-mock';
+import loginAction from '../loginAction';
+import { LOGIN_FAIL, LOGIN_SUCCESS } from '../../ActionTypes';
+
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+const loginUrl = 'https://ah-backend-xmen-staging.herokuapp.com/api/users/login/';
```

Comment:
Why don't we get this from the environment since we have it there and append the `/users/...`

Reply:
@d-rita in your `.env` there should be a `BASE_URL` that points to the backend.